ts/features/itwallet/common/utils/itwMocksUtils.ts

```diff
@@ -19,7 +19,8 @@ export enum CredentialType {
   EUROPEAN_HEALTH_INSURANCE_CARD = "EuropeanHealthInsuranceCard",
   EUROPEAN_DISABILITY_CARD = "EuropeanDisabilityCard",
   DRIVING_LICENSE = "MDL",
-  PID = "PersonIdentificationData"
+  PID = "PersonIdentificationData",
+  MDL = "org.iso.18013.5.1.mDL"
```

Comment:
I think we should rethink the concept of credential type in a Wallet where the same credential is stored in different formats (mdoc and SD-JWT). We currently use the credential type as the unique key for the cards displayed in the wallet screen: this means we would show the MDL twice to the user.

We should probably keep the same credential type for the same credential, and maybe use the `credential_configuration_id` found in the Issuer's EC (for instance `mso_mdoc_mDL`/`dc_sd_jwt_mDL`) to differentiate between variants of the same credential. This will be addressed in a future PR, I just wanted to bring the issue to attention.

Reply:
I agree with @gispada. Also I would add that `CredentialType` was born as a mock: we should try to not rely on it as much as possible.